Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -49,12 +49,12 @@ def customisePostEra_Run2_2017(process):
 
 def customisePostEra_Run2_2017_express_trackingOnly(process):
     customisePostEra_Run2_2017(process)
-    from Configuration.DataProcessing.Utils import customise_HPbeamspot
+    from Calibration.TkAlCaRecoProducers.PCLHPbeamspot_custom import customise_HPbeamspot
```

Comment:
please ```import  customise_HPbeamspot as _customise_HPbeamspot``` so that this doesn't propagate 
[although it's probably OK here because the import is inside a special method]